wallet/reference/multichain-api.md

```diff
@@ -0,0 +1,396 @@
+---
+description: See the Multichain API reference.
+sidebar_custom_props:
+  flask_only: true
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Multichain API
+
+:::flaskOnly
+:::
+
+Dapps can call Multichain API [methods](#methods) to create and manage
+[CAIP-25](https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-25.md) connections with MetaMask.
+The API also provides [events](#events) that wallets can send to dapps to notify them of onchain or connection changes.
+
+:::note See also
+- [About the Multichain API](../concepts/multichain-api.md)
+- [Interact with multiple networks](../how-to/manage-networks/use-multichain.md)
+:::
+
+## Methods
+
+### `wallet_createSession`
+
+Creates a multichain connection with a wallet, authorizing that wallet with the specified set of scopes
+and properties.
+This method is defined in [CAIP-25](https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-25.md).
+
+#### Parameters
+
+- `optionalScopes`: `object` - (Optional) [CAIP-217](https://chainagnostic.org/CAIPs/caip-217) authorization scopes the wallet can support in order to be used with this dapp.
+  If scopes are specified, only the following properties are supported:
+  - `references`: `array` - (Optional) A list of references to specific blockchains for the namespace of this scope.
```

Comment:
this can only be specified if the key of `optionalScopes` is just an ecosystem like `eip155`, so the following is valid
```
{
  'eip155': {
     references:['1', '5']  
   }
}
```

but this is not
```
{
  'eip155:1': {
     references:['1', '5']  
   }
}
```

Reply:
Added more description to clarify this. The example also shows the differences.